WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -146,9 +136,10 @@ class SiteDesignContentCollectionViewController: CollapsableHeaderViewController
 
     let selectedPreviewDevice = PreviewDevice.mobile
 
-    init(createsSite: Bool, _ completion: @escaping SiteDesignStep.SiteDesignSelection) {
-        self.completion = completion
+    init(creator: SiteCreator, createsSite: Bool, _ completion: @escaping SiteDesignStep.SiteDesignSelection) {
```

Comment:
nit: why do we want an implicit parameter for the completion? If for any reason the caller wants to include in the argument (instead of using the trailing notation) is probably clearer to explicitly use the name)

Reply:
Good catch! It [predated this PR](https://github.com/wordpress-mobile/WordPress-iOS/commit/d11154c12432350ce358f3e3bc161168dfcba96c) so I agree it should be updated.

Reply:
Updated in 1e3583985f321d4be7f8fe670082d64eafb5c4e5.